openqabot/approver.py

```diff
@@ -131,6 +132,49 @@ def is_job_marked_acceptable_for_incident(self, job_id: int, inc: int) -> bool:
             pass
         return False
 
+    @lru_cache(maxsize=512)
+    def was_ok_before(self, failed_job_id: int, inc: int) -> bool:
+        # We need a considerable amount of older jobs, since there could be many failed manual restarts from same day
+        older_jobs = self.client.get_older_jobs(failed_job_id, 20)
+
+        current_build = older_jobs['data'][0]['build'].replace('-1','')
```

Comment:
at some point of time we had two triggers during a day . Is it fully disabled currently ?  Is there chance that we will get builds ending with `-2` ?

Reply:
We don't have that anymore. We opted for more time to review failures instead of more tests left unreviewed,  In any case we don't care about that id, since we look at build date to make sure build is not too old than specified number of days and we also check that the selected job has the update under test.
We could improve the code to remove the last 2 chars in the date to be compatible, instead of looking for "-1"